### ISSUE TITLE
new crashes on the last build

### ISSUE THREAD

**Issue by unlive74** [BimmerGestalt/AAIdrive#382]
Hi, it's me again
ca351f4 crashes - in the [log](https://github.com/BimmerGestalt/AAIdrive/files/7074600/logcat_08-30-2021_11-10-32.zip) it's at 11:10. 61ad2a2 works well.

**Comment by hufman** [comment]
Thank you for the quick testing and bug report! I especially appreciate the detailed log and the pointer of where to look.
The log indicates it's crashing in the same codepath as #380, but instead of asking for `java/time/Instant` it's trying to load `j$/time/Instant`. I saw this show up in analytics and thought it was part of my (quickly-reverted) attempt to enable minification yesterday, and so I'm disappointed that it's still happening!
I wonder if it's related to 351971a? It was meant to fix java/time/Instant on API24 phones, but the analytics seem to indicate that it didn't help that anyways. Try out [this branch](https://bimmergestalt.s3.amazonaws.com/aaidrive/builds/androidautoidrive-latest-revert_desugar-nomap-sentry-release.apk) where I've reverted that commit.
